tests/test_fast_setup/test_fast_setup.py

```diff
@@ -8,10 +8,15 @@
 
 log = get_logger(__name__)
 
-mrsal = Mrsal(host=test_config.HOST,
-             port=config.RABBITMQ_PORT,
-             credentials=config.RABBITMQ_CREDENTIALS,
-             virtual_host=config.V_HOST)
+mrsal = Mrsal(
+    # host=test_config.HOST,
+    host='rabbitmq.neomodels.app',
```

Comment:
Add this to `test/config.py` to be useable in other tests

Reply:
can come from .env